install/first_google_boot.sh

```diff
@@ -222,8 +222,10 @@ else
 fi
 
 # apply outstanding updates since time of image creation
```

Comment:
Update comment here to say
Apply outstanding OS updates since time of image creation
but keep spinnaker version itself intact.